### PR TITLE
OKTA-795656 OIN Wizard: new Use Cases field and automatic use case assignment

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add new Use Cases field in OIN Wizard and automatic use case assignment
- **Is this PR related to a Monolith release?** Yes, 2024.12.0

### Resolves:

* [OKTA-795656](https://oktainc.atlassian.net/browse/OKTA-795656)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/118b6daaec2a16837c95f19fd9ece12854bfec6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2ec39623-fa76-4002-a34b-de8dadf830aa) | 43  | 83  | 76  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/118b6daaec2a16837c95f19fd9ece12854bfec6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5d78b3dc-5690-4e96-abdf-3bec1e351344) | 69  | 86  | 79  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9dc32550-e657-4aa7-b272-e957063dae71?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/659cad4169f79553e2f7a802c856933ee99ae12f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bc5bf55f-da01-4b31-a13c-01df1957b449) | 43  | 83  | 78  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/659cad4169f79553e2f7a802c856933ee99ae12f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a84783d0-a927-47bf-a61c-3d2a085bb038) | 69  | 86  | 79  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.be3e9e2a-3df5-4fd3-b597-341817b2abeb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ To access the OIN Wizard and the **Your OIN Integrations** dashboard in your org
 * You must have either the super admin or the app and org admin [roles](https://help.okta.com/okta_help.htm?type=oie&id=ext-administrators-admin-comparison) assigned to you.
 * Use your company domain email as your username for your Okta admin account (submissions from a personal email account aren't reviewed).
 
-> **Note:** The app admin role enables you to view and edit details in the OIN Wizard. To test in the OIN Wizard, if you don't have the super admin role, then you must have both the app admin and the org admin roles assigned to you.
+> **Note:** The app admin role enables you to view and edit details in the OIN Wizard. For testing in the OIN Wizard, you must have both the app admin role and the org admin role assigned to you. The super admin role gives you access to all functionality in the OIN Wizard.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:79]:
```suggestion
> **Note:** The app admin role enables you to view and edit details in the OIN Wizard. To test in the OIN Wizard, if you don't have the super admin role, then you must have both the app admin and the org admin roles assigned to you.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/bdc103f5724c266b37283ef4d3bf99531196b3dc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b2f683d6-668b-4602-90a7-1b9a50a686f8) | 42  | 83  | 77  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/bdc103f5724c266b37283ef4d3bf99531196b3dc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/60eaa2a6-575b-4df8-822b-3af6a7bdead6) | 67  | 86  | 79  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3cc21b97-0377-416d-9faa-46cdd9bef467?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/185746297e9e9e85977b0257e7a075599b602592/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/752db8d9-81a1-498e-85e9-db4fed80808b) | 42  | 87  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/185746297e9e9e85977b0257e7a075599b602592/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/41884187-f9ae-4fbc-be79-a1c1b8aa8484) | 75  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0ce6642c-c330-4f56-97fe-824921d0080f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/7904f7c4405ed182b0aa646e7926bad352b0af5e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/81a32a27-fd8d-4c3c-a901-8549ea3216d2) | 42  | 87  | 84  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md](https://github.com/okta/okta-developer-docs/blob/7904f7c4405ed182b0aa646e7926bad352b0af5e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-oin-app%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/02ab63b9-503e-405e-9da7-7b54f361cf1d) | 75  | 86  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.576a7e3a-e838-4978-a3b9-66134755fd49?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
lgtm

**Pull request merged**